src/posts/paper/14-research-sources.ts

```diff
@@ -106,7 +106,7 @@ We get some corroboration for this number from the study of train passengers, [H
 
 ### Basic Method: Underreporting factor
 
-Previously we rolled out own estimation of underreporting factor based on raw caseloads. We now aim to have our cases numbers roughly follow those estimated by [covidestim](https://covidestim.org/) - a collaboration between Yale School of Public Health  and Harvard School of Public health. At the moment we have chosen underreporting factors that approximate their numbers, but are in the process of changing our injest process to use their numbers directly.
+Previously we rolled out own estimation of underreporting factor based on raw caseloads. We now aim to have our cases numbers roughly follow those estimated by [covidestim](https://covidestim.org/) - a collaboration between Yale School of Public Health  and Harvard School of Public health. At the moment we have chosen underreporting factors that approximate their numbers, but are in the process of changing our ingest process to use their numbers directly.
```

Comment:
There are two spaces between "Health" and "and":
`Public Health  and Harvard School`

Reply:
Outdated.